ha_cluster_exporter.go

```diff
@@ -73,15 +73,33 @@ func landingpage(w http.ResponseWriter, r *http.Request) {
 		</html>`))
 }
 
+func loglevel(opt string) {
+	switch opt {
+	case "error":
+		log.SetLevel(log.ErrorLevel)
+	case "warn":
+		log.SetLevel(log.WarnLevel)
+	case "info":
+		log.SetLevel(log.InfoLevel)
+	case "debug":
+		log.SetLevel(log.DebugLevel)
+	}
+}
+
 var (
 	clock      Clock = &SystemClock{}
 	portNumber       = flag.String("port", "9002", "The port number to listen on for HTTP requests.")
+	ip4              = flag.String("ip4", "0.0.0.0", "The IP to listen on for HTTP requests.")
```

Comment:
I'm thinking here, why should we call it `ip4`? 

To me it sounds perhaps to specific. Looking at the code, we could call it `ip`.

Because if we call it `ip4` an user would expect then to have a `ip6` flag, which would just duplicate stuff.

Reply:
I would add that we shouldn't distinguish between ip4 and ip6 because `net.Dial()` (which is the socket backend eventually used by `http.ListenAndServe`) [supports both](https://golang.org/pkg/net/#Dial).
Also, I suggest calling this just `address`, because it doesn't necessarily need to be an IP, it could also be a hostname.

Reply:
Yes, that's a really good point, I'll change this.